package.json

```diff
@@ -236,13 +218,15 @@
     "flow-runtime": "^0.17.0",
     "flow-typed": "^2.4.0",
     "identity-obj-proxy": "^3.0.0",
-    "jest": "^24.8.0",
+    "jest": "^23.0.0",
```

Comment:
I know you purposely set Jest to 24.8, but it was breaking all my tests for some reason so will have to look into why on Monday.